ecs-cli/modules/commands/compose/service/compose_service_command.go

```diff
@@ -76,7 +76,7 @@ func createServiceCommand(factory composeFactory.ProjectFactory) cli.Command {
 func startServiceCommand(factory composeFactory.ProjectFactory) cli.Command {
 	return cli.Command{
 		Name:   "start",
-		Usage:  "Starts one copy of each of the containers on the created ECS service. This command updates the desired count of the service to 1.",
+		Usage:  "Starts one copy of each of the containers on the created ECS service. If the desired count is 0, it sets the desired running count to 1; otherwise this command is a no-op.",
```

Comment:
How about this:

Starts one copy of each of the containers on an existing ECS service by setting the desired count to 1 (only if the current desired count is 0).